closed/OpenJ9.gmk

```diff
@@ -73,12 +73,13 @@ define generated_target_rules
 $(foreach file,$(OPENJ9_SHARED_LIBRARIES),$(eval $(call openj9_copy_file,$2/lib$(OPENJDK_TARGET_CPU_LIBDIR)/compressedrefs/$(file),$(OUTPUT_ROOT)/vm/$(file))))
 $(foreach file,$(OPENJ9_ALT_SHARED_LIBRARIES),$(eval $(call openj9_copy_file,$2/lib$(OPENJDK_TARGET_CPU_LIBDIR)/compressedrefs/$(notdir $(file)),$(OUTPUT_ROOT)/vm/$(file))))
 $(foreach file,$(OPENJ9_MISC_FILES) $(OPENJ9_PROPERTY_FILES),$(eval $(call openj9_copy_file,$2/lib$(OPENJDK_TARGET_CPU_LIBDIR)/compressedrefs/$(file),$(OUTPUT_ROOT)/vm/$(file))))
+$(foreach file,$(OPENJ9_MISC_FILES),$(eval $(call openj9_copy_file,$2/lib/$(file),$(OUTPUT_ROOT)/vm/$(file))))
```

Comment:
The options.default file needs to stay in ```$2/lib$(OPENJDK_TARGET_CPU_LIBDIR)/compressedrefs``` otherwise the VM prints a warning ```JVMVMUT003W Options file not found``` when it starts.

Reply:
Copy options.default to j2sdk-image/jre/lib/amd64/compressedrefs
Copy OMRTraceFormat.dat and J9TraceFormat.dat to j2sdk-image/jre/lib
@pshipton please review, thank you!

Reply:
```
----- Build times -------
Start 2017-11-27 16:04:23
End   2017-11-27 16:18:03
00:00:20 corba
00:00:17 demos
00:00:33 images
00:00:16 jaxp
00:00:23 jaxws
00:02:44 jdk
00:00:34 langtools
00:00:13 nashorn
00:13:40 TOTAL
-------------------------
Finished building OpenJDK for target 'images'
root@ub14hcxvm-4:/home/openj9/openj9-openjdk-jdk8# find ./build/linux-x86_64-normal-server-release/images/j2sdk-image/ -name options.default -type f
./build/linux-x86_64-normal-server-release/images/j2sdk-image/jre/lib/amd64/compressedrefs/options.default
root@ub14hcxvm-4:/home/openj9/openj9-openjdk-jdk8# find ./build/linux-x86_64-normal-server-release/images/j2sdk-image/ -name *.dat -type f
./build/linux-x86_64-normal-server-release/images/j2sdk-image/jre/lib/tzdb.dat
./build/linux-x86_64-normal-server-release/images/j2sdk-image/jre/lib/OMRTraceFormat.dat
./build/linux-x86_64-normal-server-release/images/j2sdk-image/jre/lib/J9TraceFormat.dat
```

Reply:
Note you should also run "java -version" to ensure the the JVM starts and there aren't any warnings.